common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4012,6 +4013,15 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(32)
 
+  val WORKER_READBUFFERS_CHECK_THREAD_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.readBufferDispatcherThreadWatchdog.checkInterval")
```

Comment:
```suggestion
    buildConf("celeborn.worker.readBuffer.checkThreadInterval")
```

Reply:
I think the config name "watchdog" is more understandable to the end users.